backend/src/main/java/botobo/core/domain/workbook/Workbook.java

```diff
@@ -100,4 +105,20 @@ public int cardCount() {
     public List<Card> getCardsAsList() {
         return cards.getCards();
     }
+
+    public void updateIfCan(String name, boolean opened, Long userId) {
+        if (!user.isSameId(userId)) {
+            throw new NotAuthorException();
+        }
+        this.name = name;
+        this.opened = opened;
+    }
```

Comment:
```suggestion
    public void update(String name, boolean opened) {
        this.name = name;
        this.opened = opened;
    }
    
    public void isOwner(Long userId) {
            if (!user.isSameId(userId)) {
            throw new NotAuthorException();
        }
    }
```
update 하는 메서드의 파라미터로 유저의 ID를 받으면 뭔가 유저 ID를 수정할 수 있을 것 같은 느낌이 드네요! 유저 검증과 update를 분리해보는건 어떨까요? 그냥 이 메서드를 보고 든 생각입니다! 오즈 생각도 궁금하네요 😁

Reply:
저는 서비스 쪽에 검증 코드가 있는 것 보다는 그냥 update할때 도메인에서 검증을 해주는게 좋겠다고 생각했거덩요ㅋㅋㅋ 
근데 확실히 피케이가 말한 유저 id를 수정할 수 있을거 같다는 느낌이 들긴하네요ㅋㅋㅋ 좀만 생각해보고 수정할게용